op-node/node/server_test.go

```diff
@@ -195,6 +199,49 @@ func TestSyncStatus(t *testing.T) {
 	assert.Equal(t, status, out)
 }
 
+func TestSafeHeadAtL1Block(t *testing.T) {
+	log := testlog.Logger(t, log.LevelError)
+	l2Client := &testutils.MockL2Client{}
+	drClient := &mockDriverClient{}
+	safeReader := &mockSafeDBReader{}
```

Comment:
Calls to `.AssertExpectations(t)` on the mocks embedded in the `safeReader` and `l2Client` are missing.

Reply:
Good spot - added.